Makefile

```diff
@@ -6,6 +6,8 @@ build:
 	cargo build
 build-example:
 	cd examples/ && cargo build
+build-dialdbg:
```

Comment:
I could also add this target to the `all` target but felt it wasn't "part" of `rust-utils` in the same way as the examples.